groups/nts/ntsa/ntsa_socketoptiontype.h

```diff
@@ -92,9 +92,12 @@ struct SocketOptionType {
         /// Generate timestamps for outgoing data.
         e_TX_TIMESTAMPING = 16,
 
-        /// Allow each send operation to request copy avoidance when enqueing
+        /// Allow each send operation to request copy avoidance when enqueuing
         /// data to the socket send buffer.
-        e_ZERO_COPY = 17
+        e_ZERO_COPY = 17,
+
+        // Set the TCP congestion control algorithm.
```

Comment:
`///` to include the comment in the documentation of this enumerator by doxygen.

Reply:
Done